lib/cassanity/column_family.rb

```diff
@@ -230,6 +247,23 @@ def insert(args = {})
       })
     end
 
+    # Public: Makes it possible to asynchronously insert data into the column family.
+    #
+    # args - The Hash of arguments to pass to the argument generator
+    #        (default: {}). :column_family_name and :keyspace_name are
+    #        always included.
+    #
+    # Returns whatever is returned by executor.
+    def insert_async(args = {})
```

Comment:
I'm ok with suffixing the operation name with async. Did you consider async.insert instead? So it could be column_family.async.insert. async would return the async executor or something. Just a thought. I'm ok with this style as well I think.